services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -4,16 +4,26 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
+
 import { TextInput } from '@carbon/pictograms-react'
+import { Column, Grid } from '@carbon/react'
+import { ArrowRight, Launch } from '@carbon/react/icons'
+import clsx from 'clsx'
 import Image from 'next/image'
 import Link from 'next/link'
 import { useRouter } from 'next/router'
 import { NextSeo } from 'next-seo'
 import { useContext, useEffect } from 'react'
 
+import { Dashboard, DashboardItem } from '@/components/dashboard'
+import dashboardStyles from '@/components/dashboard/dashboard.module.scss'
 import PageBreadcrumb from '@/components/page-breadcrumb'
 import PageHeader from '@/components/page-header'
+import StatusIcon from '@/components/status-icon'
+import { framework as frameworkMap } from '@/data/framework'
```

Comment:
@mattrosno I made this change for all of the content variables. Since it seems too me that it could be an error introduced for both `type` and `status` let me know if I'm misunderstanding that in any way